index.js

```diff
@@ -42,6 +41,11 @@ program
   .description("Backup the deployed version of your app")
   .action(() => backup());
 
+program
+  .command("link-domain")
```

Comment:
I think we should use the command `domain` or `custom-domain` because that's how users will understand it best — and how we'll talk about it on the website and docs (e.g. "Remake now has custom domain support!")